perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -549,6 +549,16 @@ sub is_open {
     return exists $self->open_states->{ $self->state } ? 1 : 0;
 }
 
+=head2 is_in_progress
+
+Sees if the problem is in an open, not 'confirmed' state.
+
+=cut
+
+sub is_in_progress {
```

Comment:
Sorry, I rebased by mistake, but the tests I added can be seen at https://github.com/mysociety/fixmystreet/pull/1658/commits/a91d9562c6c0ec836f10290df779e2012f18ddec#diff-63d32f28ee766541c25f7354acd8680c